eng/common/testproxy/docker-start-proxy.ps1

```diff
@@ -31,11 +36,15 @@ catch {
     Write-Error "Please check your docker invocation and try running the script again."
 }
 
-$SELECTED_IMAGE_TAG = "1369319"
+$SELECTED_IMAGE_TAG = "1.0.0-dev20220224.1"
```

Comment:
This also works if you want to be more concise (not necessarily better):

```suggestion
$SELECTED_IMAGE_TAG = $VersionOverride ?? "1.0.0-dev20220224.1"
```

Reply:
The reason I didn't do it this way is that folks are currently pulling this value from the script. EG: Python and JS both spin up the test-proxy as part of their stuff, they parse this line of code to pull the version. 

Maybe I move it to a single file that people can just read to get the version? Then it's a breaking change, but won't ever change in the future.

Reply:
Ahh I see. Yeah with little context it seems to me like keeping the version config separate from code, especially if parsed, is a good approach if you want to change it that way.

Reply:
I  am just going to bite the bullet and deal with a breaking change now.

Reply:
@benbp I have converted to version off the file. Will patch the `js` and `python` pipelines in the sync PRs when I approve this to the next stage.

Reply:
Gave me the kick in the pants I needed.